docker/lwua-ingest/Dockerfile

```diff
@@ -32,7 +34,5 @@ ENV PATH="/lwua-py/.venv/bin:$PATH"
 # ensure we have the build folder from the poetry stage of this image
 COPY --from=poetry /lwua-py /lwua-py
 
-VOLUME /data
-VOLUME /logging
-RUN ln -s /logging /lwua-py/logging
-ENTRYPOINT ["python", "-m", "lwua"]
+RUN rm -f /lwua-py/logging && ln -s /logging /lwua-py/logging
+ENTRYPOINT ["python", "-m", "lwua"]
```

Comment:
`ENTRYPOINT  echo "sleep $INGEST_START_DELAY && python -m lwua" | sh `

Reply:
added ?env variables for scheduler delta and init waittime. This is coded into the scheduler.py instead of giving it as a variable into the entrypoint